content/en/blog/2022/knative.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Distributed tracing in Knative
+date: 2022-04-12
+author: Pavol Loffay
+---
+
+In this article, you will learn how distributed tracing works in Knative and we will explore how the OpenTelemetry project can make tracing support in this environment easier. We will explore Knative under the hood to understand what distributed tracing capabilities it provides out-of-the-box and which parts of the system need additional instrumentation.
+
+In this article we will not cover Knative fundamentals, please refer to the [Knative documentation](https://knative.dev/docs/) to get familiar with the project.
```

Comment:
nit picking, you say that you will not cover fundamentals then you give an about, this irritated me while reading, maybe you move this into "About Knative" after the basic introduction:

> [...] enable loosely coupled architectures.
>
> We will not cover (more?) Knative fundamentals ...

Reply:
agree, I am moving this to the about knative section